File: app/src/main/java/com/example/lab1/MainActivity.java
package com.example.lab1;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    private Button activityOneBtn;
    private Button activityTwoBtn;
    private Button activityThreeBtn;
    private Button activityFourBtn;

    private final View.OnClickListener activityOneListener = (v) -> {
        Intent intent = new Intent(MainActivity.this, ActivityOne.class);
        startActivity(intent);
    };

    private final View.OnClickListener activityTwoListener = (v) -> {
        Intent intent = new Intent(MainActivity.this, ActivityTwo.class);
        startActivity(intent);
    };

    private final View.OnClickListener activityThreeListener = (v) -> {
        Intent intent = new Intent(MainActivity.this, ActivityThree.class);
        startActivity(intent);
    };

    private final View.OnClickListener activityFourListener = (v) -> {
        Intent intent = new Intent(MainActivity.this, ActivityFour.class);
        startActivity(intent);
    };

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        activityOneBtn = findViewById(R.id.authActivityBtn);
        activityOneBtn.setOnClickListener(activityOneListener);

        activityTwoBtn = findViewById(R.id.profileActivity);
        activityTwoBtn.setOnClickListener(activityTwoListener);

        activityThreeBtn = findViewById(R.id.settingsActivity);
        activityThreeBtn.setOnClickListener(activityThreeListener);

        activityFourBtn = findViewById(R.id.feedbackActivity);
        activityFourBtn.setOnClickListener(activityFourListener);
    }
}

File: app/src/main/java/andrei/teplyh/lab3/MainActivity.kt
package andrei.teplyh.lab3

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val secondWorkshopButton = findViewById<Button>(R.id.secondWorkshopButton)
        secondWorkshopButton.setOnClickListener{
            val intent = Intent(this, SecondWorkshop::class.java)
            startActivity(intent)
        }

        val thirdWorkshopButton = findViewById<Button>(R.id.thirdWorkshopButton)
        thirdWorkshopButton.setOnClickListener{
            val intent = Intent(this, ThirdWorkshop::class.java)
            startActivity(intent)
        }
    }
}